src/ann_solo/reader.py

```diff
@@ -231,6 +295,233 @@ def get_version(self) -> str:
         """
         return 'null'
 
+    def _parse_sptxt_spectrum(self, raw_spectrum: str, identifier: int = None)
+        -> MsmsSpectrum:
+        """
+        Takes a raw spectrum data retrieved from an sptxt file and
+        parses it to a structured object of type MsmsSpectrum.
+
+        Parameters
+        ----------
+        raw_spectrum : string
+            The spectrum in a raw format.
+        identifier : int
+            Incremented identifier of the spectrum in the library.
+
+        Returns
+        -------
+        MsmsSpectrum
+            An MsmsSpectrum object.
+        """
+        # Create a reference variable for Class RegexpTokenizer
+        # that split raw spectrum in two chunks: metadata & spectrum
+        spectrum_tk = RegexpTokenizer('Num\s?Peaks:\s?[0-9]+\n', gaps=True,
```

Comment:
**suggestion:** See previous comment about trying to do this without the NLTK dependency.